src/dodal/plan_stubs/__init__.py

```diff
@@ -0,0 +1,3 @@
+from .wrapped import move, move_relative, set_absolute, set_relative, sleep, wait
+
+__all__ = ["set_absolute", "set_relative", "move", "move_relative", "sleep", "wait"]
```

Comment:
nit: Could the order here match the order in the imports to make it obvious everything is included?